android/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
           package="com.amazonivsreactnative">
-
+  <application>
+    <meta-data
+      android:name="com.amazonaws.ivs.player.FRAMEWORK_NAME"
```

Comment:
Minor note that I'll drop in the player rename PR, but we should align the package structure naming to that of other amazon ivs packages to ensure there isn't conflicts and package aliasing is consistent.